src/Agent.js

```diff
@@ -220,6 +230,16 @@ class Agent {
 
 	/**
 	 * Generate the params in a format valid for 'fetch'
+	 * @param {object} config		Configurations to customize the request
+	 * @param {string} config.uri			The URI to request
+	 * @param {string} config.method        	The method used to request the URI, should be in uppercase.
+	 * @param {object} [config.headers]		Key/Value pairs like `{ 'X-FOO': 'foo', X-BAR: 'bar' }` to send as headers.
+	 * @param {object} [config.data]		Arbitrary data to send as the body.
+	 * @param {Auth} [config.auth]         Authorization
+	 * @param {string|object} [config.query]	Query parameters
+	 * @param {object} [config.form]		Form fields
+	 * @param {object} [config.files]		Array of file names and file content
+	 * @param {object} [config.context]		The invocation context, describing the tool and project.
```

Comment:
It looks like there is a mismatch in spaces/tabs in some jsdocs. Can we stay consistent on the whitespace?